src/components/PersonInfo.tsx

```diff
@@ -0,0 +1,25 @@
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  person: Person;
+  search: URLSearchParams;
+};
+
+export const PersonInfo: React.FC<Props> = ({ person, search }) => {
+  const { name, slug, sex } = person;
+  const SEX_FEMALE = 'f';
```

Comment:
Don't need to create a variable for 'f', already you have a enum 
<img width="291" alt="image" src="https://github.com/user-attachments/assets/d30b537d-6270-4864-8c85-293206f61d4a">